modules/javafx.graphics/src/main/java/com/sun/javafx/tk/quantum/WindowStage.java

```diff
@@ -832,7 +832,10 @@ public void closePostponed() {
     // closed notification. This state is necessary to prevent the platform
     // window from being closed more than once.
     void setPlatformWindowClosed() {
-        platformWindow = null;
+        if(platformWindow != null) {
```

Comment:
The OpenJFX style is to put a space between `if` and `(` like so:

`if (platformWindow != null)`.

The style is not consistent over the entire source base, but I believe that's the preferred way.

We should get a style guide if there isn't one yet, and ideally a linting tool but that's not related to your PR.

Reply:
Thank you for the feedback. I changed the code accordingly.
It would be great when such things could be checked by travis.